index.js

```diff
@@ -1,20 +1,42 @@
 import {HTML5Video, Log, Events} from 'clappr'
 import shaka from 'shaka-player'
 
-const SEND_STATS_AT = 30 * 1000
-const AUTO = -1
-const SHAKA_READY = 'shaka:ready'
+const SEND_STATS_INTERVAL_MS = 30 * 1e3
+const DEFAULT_LEVEL_AUTO = -1
 
-export default class DashShakaPlayback extends HTML5Video {
-  get name() {return 'dash_shaka_playback'}
+class DashShakaPlayback extends HTML5Video {
+  static get Events () {
+    return {
+      SHAKA_READY: 'shaka:ready'
+    }
+  }
+
+  static canPlay (resource, mimeType = '') {
+    shaka.polyfill.installAll()
+    var browserSupported = shaka.Player.isBrowserSupported()
+    var resourceParts = resource.split('?')[0].match(/.*\.(.*)$/) || []
+    return browserSupported && ((resourceParts[1] === 'mpd') || mimeType.indexOf('application/dash+xml') > -1)
+  }
+
+  get name () {
+    return 'DashShakaPlayback'
```

Comment:
hey @vagnervjs , is that so? 

in that case we may have to change the name back :)

i was under the impression other plugins had camel-case names in fact. but i just checked and you seem to be right! 

sorry for this mistake. you can do a PR where you change the name. would that be ok? Thanks a lot!